lib/iex/lib/iex/introspection.ex

```diff
@@ -115,13 +126,42 @@ defmodule IEx.Introspection do
         end
         :ok
       else
-        :not_found
+        if has_callback?(mod, fun, arity) do
+          :behaviour_found
+        else
+          :not_found
+        end
       end
     else
       :no_docs
     end
   end
 
+  defp has_callback?(for, fun) do
```

Comment:
I was shocked to see `for |>`
then i realized `for` is a variable. I think it will be wise to choose a different name